### ISSUE TITLE
Cannot save notes anymore (java.io error)

### ISSUE THREAD

**Issue by fosBHY** [SimpleMobileTools/Simple-Notes#264]
As I got told in #262 there actually is a sync option. 
However since I am using it I cannot save my notes anymore. The initial save/export of the notes DOES work, but after that the file appears to be readonly and I am not able to save any changes to it, neither in simpleNotes nor in any other editor.

There is an errormessage popping up after simplenotes tries to save changes I make:
```
java.io FileNotFoundException:
Failed to open for writing:
java.io.FileNotFoundException: open failed:
EISDIR (Is a directory )
```

I saved them on my external SDCard, formated with exFat.

(the files are in the directory simplenotes is looking for)

**Comment by tibbi** [comment]
does your SD card work properly in other apps? Of course I cannot write in a readonly file.

**Comment by fosBHY** [comment]
Yes it does work properly.
The file itself gets created by simplenote with no problem. Writing into it after creation is not possible.

**Comment by fosBHY** [comment]
I would really like to help by providing infos needed. 

I played around a bit. When exporting a single note, all methods export initially. it only works to save when I use the second method **and** on internal drive. 

Oddly, the second option provided in my language (german) means something like "only export once and do not sync afterwards". Its odd, because then I can actually write into the file with other tools and it will get synchronized.. again, only on internal drive. This behaviour I would have expected from the first option. Might be a translationissue, I will try to find translationfiles to check.

Also, I had to export every note individually to be presented with sync options, bulk export didn't ask me.
